apps/ledger-live-desktop/src/newArch/features/ModularDrawer/components/SelectAccountFlow/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { memo, useCallback, useState } from "react";
+import { Observable } from "rxjs";
+import { useTranslation } from "react-i18next";
+import { CryptoOrTokenCurrency } from "@ledgerhq/types-cryptoassets";
+import { Account, AccountLike } from "@ledgerhq/types-live";
+import { WalletAPIAccount } from "@ledgerhq/live-common/wallet-api/types";
+import { SelectAccountFlowContainer, SelectorContent, AccountSelectionStep } from "./components";
+import SelectAssetFlow from "../SelectAssetFlow";
+import { Header } from "../Header";
+import { FlowStep, NavigationDirection } from "../Header/navigation";
+
+type SelectAccountDrawerProps = {
+  onAccountSelected: (account: AccountLike, parentAccount?: Account) => void;
+  currencies: CryptoOrTokenCurrency[];
+  accounts$?: Observable<WalletAPIAccount[]>;
+};
+
+export const SelectAccountFlow = memo((props: SelectAccountDrawerProps) => {
+  const { t } = useTranslation();
+  const [selectedAsset, setSelectedAsset] = useState<undefined | CryptoOrTokenCurrency>();
```

Comment:
I think the easiest would be to initialize selectedAsset with `props.currencies[0]` if there is only one currency

Reply:
Oh I only just understood you meant this in the context of your previous comment. Yeah that's a good point, i'll work on that now